### PR TITLE
Add notes for 0.50 ahead of release

### LINKED ISSUE #82: Add pre-release notes for 0.50
- [x] core
- [ ] data node

### PR DESCRIPTION
**Pull request by candida-d**
Closes #82

### CODE REVIEW

File: docs/releases/core/release-notes.md [hunk 1]
```diff
@@ -6,8 +6,250 @@ hide_title: false
...
 
-### Versions 0.46.0-0.47.6 combined | 2022-01-11
+### Versions 0.50.3-0.49.8 combined | 2022-04-27
+This release was made available to validators on 27 April, 2022.
+
+The primary focus of this and the next upcoming releases has been to complete the final remaining features, progress data-node improvements for scalability and to add test coverage and fix bugs.
+
+Note: While many of the features below are related to trading, it is not yet enabled on mainnet. 
+
+**Proposals to change market parameters**: After a market has been proposed and enacted, changes to the market parameters are required. Tokenholders will be able to submit proposals to change market parameters.
```
Comment (edd) [inline docs/releases/core/release-notes.md:16]: > "After a market has been proposed and enacted, changes to the market parameters are required."

They are? Isn't this more of an 'IF' changes are required, then...? Feel free to close if I'm wrong.

Comment (candida-d) [reply docs/releases/core/release-notes.md:16]: Not sure how that slipped by - you're right, thanks!

File: docs/releases/core/release-notes.md [hunk 2]
```diff
@@ -6,8 +6,250 @@ hide_title: false
...
+
+This data is now stored in the checkpoints, which means that it is now possible to restart from checkpoints asynchronously, which removes the synchronisation period (when feasible). This will become especially important as validator numbers increase and the network sees validators joining and leaving based on stake.
+
+**Add Ethereum key rotation support**: Vega now supports validators rotating their Ethereum keys. Ethereum keys are required so that validators can allow deposits and withdrawals via the Ethereum bridge. The controller of the bridge is a multisig bundle, and periodically validators will want to change their keys but still be part of the controlling group. This feature allows them to do this with a new transaction type.
+
+Read more: [Key management](https://github.com/vegaprotocol/specs-internal/blob/master/protocol/0067-KEYS-key_management.md)
```
Comment (edd) [inline docs/releases/core/release-notes.md:36]:
```suggestion
Read more: [Key management](https://github.com/vegaprotocol/specs/blob/master/protocol/0067-KEYS-key_management.md)
```

**Review by edd — APPROVED** [review]
✅✅✅

**Pull request merged**